### ISSUE TITLE
how to covert bit to bool

### ISSUE THREAD

**Issue by uliian** [go-xorm/xorm#673]
In mysql a field type is bit(1).how to covert bit(1) to bool?

**Comment by lunny** [comment]
```Go
type struct {
IsBool bool `xorm:"bit(1)"`
}
```
Could you define like this?

**Comment by uliian** [comment]
```go
type Group struct {
	Id      int64
	Code    string
	Name    string
	Type    int8
	Status  int8
	Deleted bool `xorm:"bit(1)"`
}

fun main(){
     groups := make([]models.Group,0);
     err := engine.Find(&groups)
}
```
erro:arg deleted as bool: strconv.ParseBool: parsing "\x00": invalid syntax

**Comment by lunny** [comment]
fixed by #712
